experimental/PropertyDDS/examples/property-inspector/package.json

```diff
@@ -46,13 +46,14 @@
 		"@fluid-experimental/property-inspector-table": "workspace:~",
 		"@fluid-experimental/property-properties": "workspace:~",
 		"@fluid-experimental/property-proxy": "workspace:~",
+		"@fluid-internal/client-utils": "workspace:~",
 		"@fluidframework/aqueduct": "workspace:~",
-		"@fluidframework/common-utils": "^1.1.1",
```

Comment:
This one doesn't seem like it used the dep at all. I'd probably skip it altogether in this PR and just remove the dep on a separate one.

Reply:
While that would be more correct, possibly, it doesn't seem necessary. Part of the intent of this change is to remove all dependencies on common-utils, even if previously they weren't used, so to do it correctly I'd need to pull this change out into a separate change and merge it before I merge this.

Reply:
And now that the new deps aren't being added to it, I agree this is fine.